common/src/main/java/com/microsoft/identity/common/internal/controllers/LocalMSALController.java

```diff
@@ -400,6 +388,47 @@ public AcquireTokenResult acquireTokenSilent(
         return acquireTokenSilentResult;
     }
 
+    private void setAcquireTokenResult(final AcquireTokenResult acquireTokenSilentResult,
```

Comment:
Can you add some comments as to why we need this method and what does it achieve?

Reply:
We don't need it..

Reply:
I moved it our of the submitSilent method to reuse the acquireTokenSilentResult.setLocalAuthenticationResult() operation.

Reply:
I believe you were going to remove this method....but I see it still being here

Reply:
Again, it's handling duplicate code.